frontend/ui/widget/pathchooser.lua

```diff
@@ -117,15 +115,15 @@ function PathChooser:onMenuHold(item)
         if self.detailed_file_info then
             local filesize = util.getFormattedSize(attr.size)
             local lastmod = os.date("%Y-%m-%d %H:%M", attr.modification)
-            title = T(_("Choose this file?\n\n%1\n\nFile size: %2 bytes\nLast modified: %3"),
+            title = T(_("Choose this file?\n\n%1\n\nFile size: %2 bytes\nLast modified: %3\n"),
```

Comment:
It's better to append all of these. Not only in general so as not to bother the translators with random newlines before or after, but especially here since these strings are already translated in their current form. (In reverse I'm open to it, i.e., changing a \n at the end to an appended \n.)